packages/devtools_app/lib/src/debugger/debugger_screen.dart

```diff
@@ -234,6 +240,51 @@ class DebuggerScreenBodyState extends State<DebuggerScreenBody>
       },
     );
   }
+
+  Widget _callStackRightChild() {
+    return CopyToClipboardControl(
+      dataProvider: () =>
+          controller.stackFramesWithLocation.value
+              .map((frame) {
+                final asyncMarker =
+                    frame.frame.kind == FrameKind.kAsyncSuspensionMarker;
+                return '${_descriptionFor(frame)}${asyncMarker ? null : ' ' + _locationFor(frame)}';
```

Comment:
use string interpolation instead of `+`: e.g. 
`'${_descriptionFor(frame)}${asyncMarker ? null : ' '}${_locationFor(frame)}'`

Also, let's make this a getter on `StackFrameAndSourcePosition`: `callStackDisplay` or similar. You can more the _descriptionFor and _locationFor helpers to the `StackFrameAndSourcePosition` class as well.